### ISSUE TITLE
cargo tarpaulin fails with weird argv error

### ISSUE THREAD

**Issue by zacchiro** [vigna/webgraph-rs#49]
```
$ cargo tarpaulin --engine=llvm
Sep 15 17:19:24.459  INFO cargo_tarpaulin::config: Creating config
Sep 15 17:19:24.555  INFO cargo_tarpaulin: Running Tarpaulin
Sep 15 17:19:24.555  INFO cargo_tarpaulin: Building project
Sep 15 17:19:24.555  INFO cargo_tarpaulin::cargo: Cleaning project
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `/home/zack/.local/share/rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustc - --crate-name ___ --print=file-names -Cdebuginfo=2 --cfg=tarpaulin -Cinstrument-coverage -Clink-dead-code -Ctarget-cpu=native -A non-camel-case-types non-upper-case-globals non_snake_case --crate-type bin --crate-type rlib --crate-type dylib --crate-type cdylib --crate-type staticlib --crate-type proc-macro --print=sysroot --print=split-debuginfo --print=crate-name --print=cfg` (exit status: 1)
  --- stderr
  error: multiple input filenames provided (first two filenames are `-` and `non-upper-case-globals`)

Sep 15 17:19:24.635 ERROR cargo_tarpaulin: Cargo failed to run! Error: cargo run failed
Error: "Cargo failed to run! Error: cargo run failed"
```

**Comment by zacchiro** [comment]
Looks this upstream bug in tarpaulin: https://github.com/xd009642/tarpaulin/issues/1307 (if so, it is fixed already in a dev branch of tarpaulin, we just need to wait.)

**Comment by zacchiro** [comment]
Actually, the release is already out. To fix this if you are affected, just upgrade tarpaulin with `cargo install cargo-tarpaulin`. It is fixed with tarpaulin 0.27.0 (at least).
